nixos/modules/services/web-apps/shiori.nix

```diff
@@ -31,6 +31,20 @@ in {
         example = "/shiori";
         description = "The root of the Shiori web application";
       };
+
+      httpSecretKey = mkOption {
+        type = types.str;
+        example = "SuperSecretPassword";
+        description =
+          "When empty all sessions will be invalidated on server restart";
+      };
+
+      databaseUrl = mkOption {
+       type = types.str;
+       default = "";
+       example = "postgresql:///shiori?host=/run/postgresql";
+       description = "The connection URL to connect to MySQL or PostgreSQL";
+      };
```

Comment:
Is this a required connection? In that case, I think you should drop the `default`, so that the user will be warned when it is left unset.

Reply:
It's not a required connection, if it's unset, shiori will create a sqlite database.